src/SConscript

```diff
@@ -615,7 +615,7 @@ PySource('m5', 'python/m5/defines.py')
 
 # Generate a file that wraps the basic top level files
 gem5py_env.Command('python/m5/info.py',
-            [ File('#/COPYING'), File('#/LICENSE'), File('#/README'),
+            [ File('#/COPYING'), File('#/LICENSE'), File('#/README.md'),
```

Comment:
This is OK for now. However, I think we should just remove `info.py` completely. I can't think of any other tool that has the readme/license embedded this way.

Reply:
> This is OK for now. However, I think we should just remove `info.py` completely. I can't think of any other tool that has the readme/license embedded this way.

I'm not sure I know exactly why we do this at all. I assume we print this to a terminal somewhere? Any way, seems weird. I'll put it in an issue.

Reply:
Issue #89 is to remove this at some point.